Sources/MapboxNavigation/NavigationView.swift

```diff
@@ -172,9 +172,9 @@ open class NavigationView: UIView {
     
     lazy var speedLimitView: SpeedLimitView = .forAutoLayout(hidden: true)
     
-    lazy var topBannerContainerView: BannerContainerView = .forAutoLayout()
+    public lazy var topBannerContainerView: BannerContainerView = .forAutoLayout()
```

Comment:
Is it safe to provide readwrite access to these properties?

Reply:
Idea is to allow users use `NavigationView` independently. This change will allow them to modify top and bottom banners.